Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertConnector.cs

```diff
@@ -22,18 +22,18 @@ public RevitMEPConnector ConnectorToSpeckle(Connector connector)
     {
       speckleMEPConnector.angle = connector.Angle;
     }
-    catch { }
+    catch (Autodesk.Revit.Exceptions.InvalidOperationException) { }
```

Comment:
If these are well known errors that we can safely recover from, could we at least have a comment explaining why is this ok on each case?

Could this be prevented if we re-wrote this logic a bit?

Reply:
yes, refactored